CONTRIBUTING.md

```diff
@@ -11,16 +20,39 @@ If you are Algolia employee and want to manage a DocSearch account,
 you'll need to add the following variables in your `.env` file:
 
 ```
+APPLICATION_ID=
+API_KEY=
+APPLICATION_ID_PROD=
+API_KEY_PROD=
+HELP_SCOUT_API_KEY=
+NB_HITS_MAX=
+
 WEBSITE_USERNAME=
 WEBSITE_PASSWORD=
 SLACK_HOOK=
 SCHEDULER_USERNAME=
 SCHEDULER_PASSWORD=
+MONITORING_API_KEY=
+PUBLIC_CONFIG_FOLDER=<path to your docsearch-scrapy directory> + /configs/public
```

Comment:
Shouldn't this be:
```suggestion
PUBLIC_CONFIG_FOLDER=<path to your docsearch-scraper directory> + /configs/public
```

Reply:
Also, if it's always a subdirectory of the scraper, we can guess the filepath easily, without the need for a variable, right?

Reply:
Yes